components/content-blocks/SummitStatus/index.js

```diff
@@ -15,7 +16,9 @@ const SummitStatus = ({ summitStatusLayout, widgetPreviews = [] }) => {
   const [windspeedUnit, setWindspeedUnit] = useState(
     language === "en" ? "NM" : "m"
   );
-  const [tempUnit, setTempUnit] = useState(language === "en" ? "F" : "C");
+  const [tempUnit, setTempUnit] = useState(
+    language === "en" ? "fahrenheit" : "celsius"
+  );
```

Comment:
JavaScript's number format library uses the long strings and convert.js can also use long strings so I reverted to those